src/components/WeatherWidgetMessage.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="wrapper">
```

Comment:
Since it's not just a wrapper but the wrapper inside WeatherwidgetMessage, let's rename the class to weather-message-wrapper or even weather-status-wrapper or status-message-wrapper. Up to you.